javascript/javascript1/week1/week1.js

```diff
@@ -6,63 +6,101 @@ const age = yearFuture - yearOfBirth;
+const peterHouseDepth = 10;
+const peterHouseHeight = 10;
+const peterGardenSize = 100;
+const peterAmountPaid = 2500000;
 
-const peterHousePrice = peterVolumeInMeters * 2.5 * 1000 + peterHouseDetails[3] * 300;
-const juliaHousePrice = juliaVolumeInMeters * 2.5 * 1000 + juliaHouseDetails[3] * 300;
+const juliaHouseWidth = 5;
+const juliaHouseDepth = 11;
+const juliaHouseHeight = 8;
+const juliaGardenSize = 70;
+const juliaAmountPaid = 1000000;
+const peterVolumeInMeters =
+  peterHouseWidth * peterHouseDepth * peterHouseHeight;
+const juliaVolumeInMeters =
+  juliaHouseWidth * juliaHouseDepth * juliaHouseHeight;
 
-if (peterHousePrice>peterHouseDetails[4]){
-    console.log("Peter, You bought this house for a cheaper price");
-}
-else if(peterHousePrice===peterHouseDetails[4]){
-    console.log("Peter, You bought this house for correct price");
-}
-else{
-    console.log("Peter, You bought this house for higher price");
-}
-if (juliaHousePrice>juliaHouseDetails[4]){
-    console.log("Julia, You bought this house for a cheaper price");
-}
-else if(juliaHousePrice===juliaHouseDetails[4]){
-    console.log("Julia, You bought this house for correct price");
+const peterHousePrice =
+  peterVolumeInMeters * 2.5 * 1000 + peterGardenSize * 300;
+const juliaHousePrice =
+  juliaVolumeInMeters * 2.5 * 1000 + juliaGardenSize * 300;
+
+if (peterHousePrice > peterAmountPaid) {
+  console.log("Peter, You bought this house for a cheaper price");
+} else if (peterHousePrice === peterAmountPaid) {
+  console.log("Peter, You bought this house for correct price");
+} else {
+  console.log("Peter, You bought this house for higher price");
 }
-else{
-    console.log("Julia, You bought this house for higher price");
+if (juliaHousePrice > juliaAmountPaid) {
+  console.log("Julia, You bought this house for a cheaper price");
+} else if (juliaHousePrice === juliaAmountPaid) {
+  console.log("Julia, You bought this house for correct price");
+} else {
+  console.log("Julia, You bought this house for higher price");
 }
 
 //Ez Namey (Startup name generator) Optional
-console.log("************ Ez Namey (Startup name generator) Optional *****************");
-const firstWords = ["Easy", "Awesome", "Corporate", "NextGen", "Innovative", "Creative", "CodeCraft", "Hackers", "Digital", "IT" ];
-const secondWords = ["Solutions", "World", "Themes", "Hands", "Providers", "Services", "Group", "Waves", "Ground", "Players"];
+console.log(
+  "************ Ez Namey (Startup name generator) Optional *****************"
+);
+const firstWords = [
+  "Easy",
+  "Awesome",
+  "Corporate",
+  "NextGen",
+  "Innovative",
+  "Creative",
+  "CodeCraft",
+  "Hackers",
+  "Digital",
+  "IT",
+];
+const secondWords = [
+  "Solutions",
+  "World",
+  "Themes",
+  "Hands",
+  "Providers",
+  "Services",
+  "Group",
+  "Waves",
+  "Ground",
+  "Players",
+];
 const firstName = firstWords[Math.floor(Math.random() * 10)];
 const secondName = secondWords[Math.floor(Math.random() * 10)];
```

Comment:
Suggestion: It can be more precise to add to the variable name that the result is random eg. `randomFirstName`.